src/components/UserCard/UserCard.js

```diff
@@ -83,20 +83,37 @@ const UserCard = props => {
     [css.withBioMissingAbove]: !hasBio,
   });
 
-  const hideContact = currentUser && isCurrentUser;
-  const separator = hideContact ? null : <span className={css.linkSeparator}>•</span>;
-  const contact = hideContact ? null : (
+  const separator = isCurrentUser ? null : <span className={css.linkSeparator}>•</span>;
```

Comment:
The intended behaviour is:
- unauthenticated user: `• contact`
- authenticated user, but not author: `• contact`
- authenticated user, author: `• edit`

If I understood this right, the separator should be printed always (no need for ifs anymore)

Also, `{contact}` and `{editProfileMobile}` are alternatives for each others. So, for the clarity: 
 => `{isCurrentUser ? editProfileMobile : contact}`

Reply:
Almost, but as seen in the second screenshot from the top, in the desktop view the edit profile link is on the right and the separator is hidden.